design/templates/solution_design.md

```diff
@@ -67,5 +71,41 @@
 ## Open questions
 [//]: # (Add any question that is still open. It makes it easier for the reader to have the open questions accumulated here istead of them being acattered along the doc)
 
-## Implementation plan
+## Delivery plan
 [//]: # (Break the solution into tasks)
+
+### Total
+[//]: # (Write the total estimation)
+
+### Parallelism
+[//]: # (Suggest how many team members can work on the solution in parallel)
+
+### Ramp up
+[//]: # (Estimate the tasks)
+
+### Infrastructure and preparations
```

Comment:
maybe this should just be "Infrastructure Requirements"? or do you intend for it just to be any new infra that'll be needed for the task?

Reply:
Yes, it is not just requirements for infra teams

Reply:
@nessiLahav can you please elaborate in the description? what do you expect the template user to add here?

Reply:
Done :)